dask/dataframe/core.py

```diff
@@ -2236,10 +2237,15 @@ def min(self, axis=0, skipna=True, split_every=False, out=None, numeric_only=Non
         )
 
     @derived_from(pd.DataFrame)
-    def idxmax(self, axis=None, skipna=True, split_every=False):
+    def idxmax(
+        self, axis=None, skipna=True, split_every=False, numeric_only=no_default
+    ):
         fn = "idxmax"
         axis = self._validate_axis(axis)
-        meta = self._meta_nonempty.idxmax(axis=axis, skipna=skipna)
+        numeric_only_kwargs = check_numeric_only_valid(numeric_only, "idxmax")
+        meta = self._meta_nonempty.idxmax(
+            axis=axis, skipna=skipna, **numeric_only_kwargs
+        )
         if axis == 1:
             return map_partitions(
                 M.idxmax,
```

Comment:
Should this be `_getattr_numeric_only` instead?

Reply:
Good point, have to check

Reply:
No the keyword was added in pandas 1.5 with the proper default, so there wasn't a deprecation message